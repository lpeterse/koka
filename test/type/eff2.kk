fun twice(f,x) {
  return f(f(x))
}

fun twice2(f,x) {
  return id(f(id(f(x))))
}