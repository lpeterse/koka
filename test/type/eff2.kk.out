twice      : forall<a,e> (f : (a) -> e a, x : a) -> e a
twice2     : forall<a,e> (f : (a) -> e a, x : a) -> e a
